src/actions/LocalSettingsActions.ts

```diff
@@ -149,15 +149,15 @@ export const writeSpendingLimits = async (account: EdgeAccount, spendingLimits:
 }
 
 export const readLocalSettings = async (account: EdgeAccount): Promise<LocalAccountSettings> => {
-  return await account.localDisklet
-    .getText(LOCAL_SETTINGS_FILENAME)
-    .then(JSON.parse)
-    .catch(async () => {
-      // If Settings.json doesn't exist yet, create it, and return it
-      const defaults = asLocalAccountSettings({})
-      return await writeLocalSettings(account, defaults).then(() => defaults)
-    })
-    .then(settings => asLocalAccountSettings(settings))
+  try {
+    const text = await account.localDisklet.getText(LOCAL_SETTINGS_FILENAME)
+    const json = await JSON.parse(text)
```

Comment:
`JSON.parse` isn't async, so no need to await. 

Just FYI, `asLocalAccountSettings` could be a codec cleaner so we could `asLocalAccountSettings(text)` here if it were to use `asJSON`. Then on the write back to disk `wasLocalAccountSettings(data)`. But that's optional.

Reply:
Good catch on the async. Will fix. Do we really need an uncleaner if the data is always type checked before write

Reply:
Uncleaner is optional. Not necessary. It just moves the JSON serialization into the cleaner.